bedrock/firefox/redirects.py

```diff
@@ -293,21 +293,14 @@ def firefox_channel(*args, **kwargs):
              'https://support.mozilla.org/kb/push-notifications-firefox'),
 
     # Bug 1239960
-    redirect(r'^firefox/partners/?$', 'firefox.os.index'),
+    redirect(r'^firefox/partners/?$', 'https://support.mozilla.org/products/firefox-os'),
 
     # Bug 1243060
     redirect(r'^firefox/tiles/?$',
              'https://support.mozilla.org/kb/about-tiles-new-tab'),
 
-    # Bug 1239863
-    redirect(r'^firefox/os/1\.1/?$', 'firefox.os.index'),
-    redirect(r'^firefox/os/1\.3/?$', 'firefox.os.index'),
-    redirect(r'^firefox/os/1\.3t/?$', 'firefox.os.index'),
-    redirect(r'^firefox/os/1\.4/?$', 'firefox.os.index'),
-    redirect(r'^firefox/os/2\.0/?$', 'firefox.os.index'),
-    redirect(r'^firefox/os/2\.5/?$', 'firefox.os.index'),
-    redirect(r'^firefox/os/faq/?$',
-             'https://support.mozilla.org/products/firefox-os'),
+    # Bug 1239863, 1329931
+    redirect(r'^firefox/os(?:/.*)?$', 'https://support.mozilla.org/products/firefox-os'),
```

Comment:
Please change this line to `redirect(r'^firefox/os(/.*)?$', 'https://support.mozilla.org/products/firefox-os'),`